README.md

```diff
@@ -93,63 +112,172 @@ const strapiConfig = {
       },
     },
   ],
+  // ...
 };
 ```
 
-#### Internationalization support
-
-Strapi now supports [internationalization](https://strapi.io/documentation/developer-docs/latest/development/plugins/i18n.html#installation). But by default, this plugin will only fetch data in the default locale of your Strapi app. If your content types are available in different locales, you can also pass an entity definition object to specify the locale you want to fetch for a content type. Use the `all` value to get all available locales on a collection type.
-
 #### Draft content
 
 Strapi now supports [Draft and publish](https://strapi.io/documentation/developer-docs/latest/concepts/draft-and-publish.html#draft-and-publish), which allows you to save your content as a draft and publish it later. By default, this plugin will only fetch the published content.
```

Comment:
The link to D&P docs is broken, we should replace it: https://docs.strapi.io/developer-docs/latest/concepts/draft-and-publish.html